components/Navbar.vue

```diff
@@ -326,11 +319,14 @@ export default class NavbarMenu extends mixins(
     clearSession()
   }
 
+  get isMobileMenuOpen() {
+    const navbarMenu = document.querySelector('.navbar-menu')
+    return navbarMenu?.classList.contains('is-active')
+  }
+
   @Watch('isBurgerMenuOpened')
   onBurgerMenuOpenedChanged() {
-    document.documentElement.style.overflow = this.isBurgerMenuOpened
-      ? 'hidden'
-      : 'auto'
+    this.toggleBodyScroll()
```

Comment:
I don't agree to add `toggleBodyScroll` everywhere. We even could not know whether it will turn on the scroll or not.